assets/js/util.js

```diff
@@ -39,6 +40,18 @@ if (typeof Object.assign != 'function') {
 }
 /* eslint-enable */
 
+function debounce(func, debounceDuration) { // eslint-disable-line no-unused-vars
+    var clock = null;
+    debounceDuration = debounceDuration || DEFAULT_DEBOUNCE_DURATION;
```

Comment:
Reassigning function parameters is generally recommend against. I would suggest just inlining this disjunction inside the `setTimeout` call.